arbitrum-docs/arbitrum-bridge/03-troubleshooting.md

```diff
@@ -7,7 +7,7 @@ content_type: troubleshooting
 ---
 
 import FAQStructuredDataJsonLd from '@site/src/components/FAQStructuredData';
-import FAQQuestions from '../partials/_troubleshooting-bridging-partial.md';
+import FAQQuestions from '../partials/_troubleshooting-bridging-partial.mdx';
```

Comment:
These are autogenerated files in `notion-docs/scripts/update.ts` so you'll have to modify the filenames there as well.